modelina-website/src/components/playground/Sidebar.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { IoOptionsOutline } from 'react-icons/io5';
+import { VscListSelection } from 'react-icons/vsc';
+import { Tooltip } from './Tooltip';
+import { usePlaygroundContext } from '../contexts/PlaygroundContext';
+
+interface SidebarItem {
+  name: string;
+  title: string;
+  isActive: boolean;
+  onClick: () => void;
+  icon: React.ReactNode;
+  tooltip: React.ReactNode;
+}
+
+interface SidebarProps {}
+
+export const Sidebar: React.FunctionComponent<SidebarProps> = () => {
+  const { setShowOptions, setShowOutputNavigation } = usePlaygroundContext();
+  const sidebarItems: SidebarItem[] = [
+    // Options
+    {
+      name: 'options',
+      title: 'Options',
+      isActive: false,
+      onClick: () => {
+        setShowOptions((prevShowOptions) => !prevShowOptions);
+      },
+      icon: <IoOptionsOutline className="w-5 h-5" />,
+      tooltip: 'Option'
```

Comment:
Think we should add a little more explanation here 🤔 
```suggestion
      tooltip: 'Show or hide all the options'
```

Feel free to go with another text if you have one in mind!